src/GHC/Util/HsPat.hs

```diff
@@ -1,9 +1,9 @@
 {-# LANGUAGE MultiParamTypeClasses , FlexibleInstances, FlexibleContexts #-}
```

Comment:
Why the rename?

Reply:
Not a technical reason. The naming scheme for these modules is that they are named after the (module/)datatype for which they relate e.g. `GHC.Util.HsExpr` for `HsExpr`, `GHC.Util.HsDecl` for `HsDecl` and so on.  Actually, reviewing this I see I that in fact, though the module name is `HsPat`, the type is now `Pat` (doh!) so it was right the way it was. I'll revert the rename.

All the above said, don't know if you are aware - on HEAD there is lots of churn here to introduce a new "GHC module hierarchy" and everything is getting moved.